tools/alive-tv.cpp

```diff
@@ -835,497 +912,634 @@ class arm2alive_ {
   }
 
   std::string next_name() {
-    return "%" + std::to_string(curInst.getOperand(0).getReg()) + "_" + std::to_string(next_id());
+    return "%" + std::to_string(curInst.getOperand(0).getReg()) + "_" +
+           std::to_string(next_id());
   }
 
   void add_identifier(IR::Value &v) {
     mc_add_identifier(curInst.getOperand(0), curId, v);
   }
 
+  // store will store an IR value using the current instruction's destination
+  // register.
+  // All values are kept track of in their full-width counterparts to simulate
+  // registers. For example, a x0 register would be kept track of in the bottom
+  // bits of w0. Optionally, there is an "s" or signed flag that can be used
+  // when writing smaller bit-width values to half or full-width registers which
+  // will perform a small sign extension procedure.
+  void store(IR::Value &v, bool s = false) {
+    // if v.bits() == 64, regSize == 64 because of above assertion
+    if (v.bits() == 64 || DISABLE_REGISTER_LOOKUP) {
+      add_identifier(v);
+      return;
+    }
+
+    // FIXME: get register bit width for further computation
+    size_t regSize = 32;
```

Comment:
Is this a typo or still a fixme? I see that bitwidth checks in other parts of this function